product_docs/docs/pem/9/registering_agent.mdx

```diff
@@ -23,42 +23,47 @@ On Windows, the PEM agent graphical installer allows you to register the agent w
+| `--allow_server_restart`        | Enable the `allow-server_restart` parameter to allow PEM to restart the monitored server. The default value is `True`.                                                                              |
+| `--allow-batch-probes`          | Enable the `allow-batch-probes` parameter to allow PEM to run batch probes on this agent. The default value is `False`.                                                                             |
 | `--batch-script-user`           | The operating system user to use for executing the batch/shell scripts. The default value is none. The scripts don't execute if you leave this parameter blank or the specified user doesn't exist. |
-| `--enable-heartbeat-connection` | Enable the `enable-heartbeat-connection` parameter to create a dedicated heartbeat connection between the PEM agent and server to update the active status. The default value is `False`.                                                  |
-| `--enable-smtp`                 | Enable the `enable-smtp parameter` to allow the PEM agent to send the email on behalf of the PEM server. The default value is `False`.                                                                                                  |
-| `--enable-snmp`                 | Enable the `enable-snmp parameter` to allow the PEM agent to send the SNMP traps on behalf of the PEM server. The default value is `False`.                                                                                             |
-| `-o`                            | Specifies whether you want to override the configuration file options.                                                                                                                                                                    |
+| `--enable-heartbeat-connection` | Enable the `enable-heartbeat-connection` parameter to create a dedicated heartbeat connection between the PEM agent and server to update the active status. The default value is `False`.           |
+| `--enable-smtp`                 | Enable the `enable-smtp parameter` to allow the PEM agent to send the email on behalf of the PEM server. The default value is `False`.                                                              |
+| `--enable-snmp`                 | Enable the `enable-snmp parameter` to allow the PEM agent to send the SNMP traps on behalf of the PEM server. The default value is `False`.                                                         |
+| `--enable-probe`                | Enable the probe using `--probe-id`, `probe-internal-name` or `probe-display-name`.                                                                                                                 |
+| `--disable-probe`               | Disable the probe using `--probe-id`, `probe-internal-name` or `probe-display-name`.                                                                                                                |
```

Comment:
These two new parameters are in PEM 9.6 (not released yet) so I suggest we remove these two lines until then. When we update for PEM 9.6 we should add a full example for this usage.